### PR TITLE
Display and save quiz duration

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

The quiz will be timed and the time will be used to calculate score bonus points. This PR sets up the timer, with lightly attempted styling, and saves it to redux to be used in #72 

Connects #60 

### Demo
Screen Shot 2019-07-01 at 1 38 56 PM
Screen Shot 2019-07-01 at 1 39 01 PM

Just so you can see the minutes properly formatted:
Screen Shot 2019-07-01 at 12 42 13 PM

## Testing Instructions

`vagrant up`
`./scripts/server`
http://localhost:3474/
Play the quiz and see the timer~

### CODE REVIEW

File: src/app/src/components/Timer.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import Text from './Text';
+import { saveSecondsToCompleteQuiz } from '../actions';
+
+const StyledTimer = styled(Box)`
+    position: absolute;
+    background-image: linear-gradient(
+        to right,
+        ${themeGet('colors.teals.0')} 30%,
+        ${themeGet('colors.teals.1')} 70%,
+        ${themeGet('colors.teals.2')} 100%
```
Comment (caseycesari) [inline src/app/src/components/Timer.js:15]: This is cool

File: src/app/src/components/Timer.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        const minutes = Math.floor(elapsedSeconds / 60);
+        const seconds = elapsedSeconds % 60;
+
+        const formattedMins = ('0' + minutes).slice(-2);
+        const formattedSecs = ('0' + seconds).slice(-2);
+
+        return (
+            <StyledTimer>
+                <Text variant='large'>
+                    {formattedMins}:{formattedSecs} min
```
Comment (caseycesari) [inline src/app/src/components/Timer.js:61]: Do we want to start the label as `secs` and then switch to `min` once the timer hits `1:00`? I was a little confused at first.

**Review by caseycesari — APPROVED** [review]
👍

File: src/app/src/components/Timer.js (hunk 2 above)
Comment (fungjj92) [reply src/app/src/components/Timer.js:61]: That's a good question. I had been following the wires, but realize with your question that there's only examples of mm:ss off which I'd made this assumption. Calling in design! @alexelash  

Screen Shot 2019-07-02 at 9 20 41 AM

File: src/app/src/components/Timer.js (hunk 1 above)
Comment (fungjj92) [reply src/app/src/components/Timer.js:15]: right? though i swiped it from another component that @alexelash designed, so i can't take any credit

File: src/app/src/components/Timer.js (hunk 2 above)
Comment (fungjj92) [reply src/app/src/components/Timer.js:61]: Alex is on vacation through RRP this week, so I'm going to merge this PR and add this comment to the Quiz style pass list.

**Pull request merged**